### DISCUSSION TITLE
Usage with moduleResolution "node"?

### DISCUSSION THREAD

**Discussion by blarfoon** [inokawa/virtua#589] [Q&A] [answered]
Is it possible to use this library in a project with moduleResolution set to "node"?

When trying to import it I get the following error

```
Cannot find module 'virtua/solid' or its corresponding type declarations.
  There are types at '...../node_modules/virtua/lib/solid/index.d.ts', but this result could not be resolved under your current 'moduleResolution' setting. Consider updating to 'node16', 'nodenext', or 'bundler'.
  ```
  
  Unfortunately I am not able to change the module resolution as almost all other libraries I use rely on it set to "node". Is there a solution/workaround?

**Comment by inokawa** [comment, marked as answer]
Officially `moduleResolution: "node"` is not supported. It's recommended to migrate to `moduleResolution: "bundler"`.

**Reply by blarfoon** [reply]
I will look into what changes that requires on my end. Thank you
